### PR TITLE
Adding additional variable options for user

### PR DESCRIPTION
**Pull request by annumberhocker**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -96,6 +95,53 @@ variable "ibmcloud_api_key" {
   description = "The api key for IBM Cloud access"
 }
 
+variable "osdStorageClassName" {
+  description = "Storage class that you want to use for your OSD devices"
+  type = string
+  default = "ibmc-vpc-block-10iops-tier"
+}
+
+variable "osdSize" {
```
Comment (timroster) [inline variables.tf:104]: @annumberhocker can you set the default to 250Gi

**Review by timroster — CHANGES_REQUESTED** [review]

File: variables.tf [hunk 2]
```diff
@@ -96,6 +95,53 @@ variable "ibmcloud_api_key" {
...
+variable "monSize" {
+  description = "Size of the storage devices that you want to provision for the monitor pods. The devices must be at least 20Gi each"
+  type = string
+  default = "20Gi"
+}
+
+variable "monStorageClassName" {
+  description = "Storage class to use for your Monitor pods. For VPC clusters you must specify a block storage class"
+  type = string
+  default = "ibmc-vpc-block-10iops-tier"
```
Comment (timroster) [inline variables.tf]: the current default SC for the add-on is now `ibmc-vpc-block-metro-10iops-tier`

**Comment by timroster** [comment]
@triceam - updates are in for the defaults now used by the add-on

**Review by timroster — APPROVED** [review]
lgtm

**Comment by annumberhocker** [comment]
Having to restructure, so closing this pull request

**Pull request closed without merging**